packages/liveblocks-core/src/room.ts

```diff
@@ -667,18 +678,13 @@ type RoomState<
     storageOperations: Op[];
   };
 
-  /**
-   * Number of reconnect() retries.
-   */
-  numRetries: number;
   readonly timers: {
+    // XXX Should we move this into the `buffer` structure, if _that_ is what
+    // is being flushed, close to the `lastFlushedAt` value?
```

Comment:
probably, especially since this is now the loneliest timer in the timers object.